api/rest/restcore/issues_rest.php

```diff
@@ -370,3 +376,52 @@ function rest_issue_monitor_add( \Slim\Http\Request $p_request, \Slim\Http\Respo
 	return $p_response->withStatus( HTTP_STATUS_CREATED, "Users are now monitoring issue $t_issue_id" )->
 		withJson( array( 'issues' => array( $t_issue ) ) );
 }
+
+/**
+ * Attach a tag to an issue.
+ *
+ * @param \Slim\Http\Request $p_request   The request.
+ * @param \Slim\Http\Response $p_response The response.
+ * @param array $p_args Arguments
+ * @return \Slim\Http\Response The augmented response.
+ */
+function rest_issue_tag_attach( \Slim\Http\Request $p_request, \Slim\Http\Response $p_response, array $p_args ) {
+	$t_issue_id = $p_args['id'];
+	$t_data = array(
+		'query' => array( 'issue_id' => $t_issue_id ),
+		'payload' => $p_request->getParsedBody(),
+	);
+
+	$t_command = new TagAttachCommand( $t_data );
+	$t_command->execute();
+
+	$t_issue = mc_issue_get( /* username */ '', /* password */ '', $t_issue_id );			
+
+	return $p_response->withStatus( HTTP_STATUS_CREATED, "Tag attached to issue $t_issue_id" )->
+		withJson( array( 'issues' => array( $t_issue ) ) );
```

Comment:
This is consistent with the case where a user adds a note, relationship, etc.  In case of such write operations, I optimized for providing the full issue for the client to get a complete accurate view of the issue rather than having to patch locally.  In the case of adding a tag, this is trivial, but in cases like adding notes, this can have side effects like changing status from feedback to assigned, or some other side effects.  Hence, I don't think it is a win to reduce the payload in case of write operations.